lib/services/android-project-service.ts

```diff
@@ -517,12 +517,18 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 		if (this.$androidToolsInfo.getToolsInfo().androidHomeEnvVar) {
 			const gradlew = this.$hostInfo.isWindows ? "gradlew.bat" : "./gradlew";
 
+			const localArgs = [...gradleArgs];
+			if (this.$logger.getLevel() === "INFO") {
+				localArgs.push("--quiet");
+				this.$logger.info(`${gradlew} ${localArgs.join(" ")}`);
```

Comment:
I prefer using `Gradle build...`
In case you want to print dots, you can use:
```TypeScript
const action = () => this.spawn(gradlew, localArgs, childProcessOpts, spawnFromEventOptions);
this.$logger.printInfoMessageOnSameLine("Gradle build...");
await this.$progressIndicator.showProgressIndicator(action(), 2000);
```